tests/functional_small/ops/ops_epetra_multi_vector.cc

```diff
@@ -79,6 +79,12 @@ TEST_F(epetraMultiVectorGlobSize15Fixture, multi_vector_update1_a)
       EXPECT_DOUBLE_EQ(v[j][i], 2.);
      }
     }
+    pressio::ops::update(v, a, 1.);
```

Comment:
similar to above

Reply:
thanks, fixed on c842063a43a2acdb082b3bb7ce17561771a4083b